### PR TITLE
Fixes issue #2334 Pass an encoded hex email to DEL /emails

### PR DESCRIPTION
**Pull request by deeptibaghel**
Fixes issue #2334 Pass an encoded hex email to DEL /emails

### CODE REVIEW

File: lib/db.js [hunk 1]
```diff
@@ -431,7 +431,7 @@ module.exports = (
 
   DB.prototype.setPrimaryEmail = function (uid, email) {
     log.trace({op: 'DB.accountFromEmail', email: email})
-    return this.pool.post('/email/' + Buffer(email, 'utf8').toString('hex') + '/account/' + uid)
+    return this.pool.post('/email/' + Buffer.from(email, 'utf8').toString('hex') + '/account/' + uid)
```
Comment (vbudhram) [inline lib/db.js]: You should be able to use `hexEncode` helper here.

**Comment by vladikoff** [comment]
@deeptibaghel , hey, make sure to change this SHA in here https://github.com/mozilla/fxa-auth-server/blob/master/npm-shrinkwrap.json#L343 to `cc6e08b7d6aba6b15d06f34ee5efe0c15bc1d72b`  (that is your latest commit SHA from https://github.com/mozilla/fxa-auth-db-mysql/commits/master )

then tests should pass

**Comment by deeptibaghel** [comment]
@vladikoff , i changed it but then the tests keep waiting forever :( , also the npm-shrinkwrap.json files shows as conflict , which is not clear to me.

**Comment by vladikoff** [comment]
> @vladikoff , i changed it but then the tests keep waiting forever :( , also the npm-shrinkwrap.json files shows as conflict , which is not clear to me.

Might have to `git rebase` against `master`. but first try to run `npm run shrinkwrap` and commit that, see if there is a conflict that way

**Pull request closed without merging**